internal/templatefn/fn.go

```diff
@@ -57,6 +57,14 @@ func All(siriusPublicURL, prefix, staticHash string) map[string]interface{} {
 
 			return time.Format(format), nil
 		},
+		"parseDateWithTime": func(s string, format string) (string, error) {
```

Comment:
I think "formatDateWithTime" would be clearer.

(A can of worms here: really we should make `createdDate` a `time.Time` instance when unmarshalling the JSON, but because it returns in "Sirius date time format" we'd need to write a custom unmarshaller. The API should be using standardised date formats but we've got too much dependent code to just switch that now 😭)